csrgen.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+import argparse
+import fnmatch
+import re
+import os
+import subprocess
+import readline
+from jinja2 import Template, Environment, FileSystemLoader
+
+file_loader = FileSystemLoader('.')
+env = Environment(loader=file_loader)
+
+
+description = """
```

Comment:
PEP8 says constants should be named in all caps.

https://peps.python.org/pep-0008/#constants

Reply:
NIT: mixed use of single-quote and double-quotes throughout this file.